packages/auto-approve/src/utils-for-pr-checking.ts

```diff
@@ -181,3 +181,18 @@ export function checkFilePathsMatch(
   }
   return filesMatch;
 }
+
+/**
+ * Checks whether the attempted merge is happening during the week.
+ *
+ * @returns true if merge is happening on the weekday, false if it's on Friday or the weekend.
+ */
+export function mergesOnWeekday(): boolean {
+  const date = new Date(Date.now());
```

Comment:
My guess is that the date is in UTC timezone, but I may be wrong. Can you confirm?
If it in UTC, you should convert to pacific time zone, right?

Reply:
I suggest using dayjs with the timezone extension:

https://day.js.org/docs/en/timezone/timezone

Works something like this:

`dayjs(new Date()).tz('Australia/ACT').$D`.

Reply:
Thanks for that tip!